### PR TITLE
Add kateolenya on map more descriptions

### PR DESCRIPTION
**Pull request by kateolenya**
Added

### CODE REVIEW

**Comment by I-litv** [comment]
HI there

File: _pins/kateolenya.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHanlde: kateolenya
```
Comment (hectorsector) [inline _pins/kateolenya.json]: @kateolenya looks like `githubHandle` is misspelled! Once you fix that you should be all set 👍

**Comment by hollenberry** [comment]
Hey @kateolenya  ! You're so close to finishing this PR! As noted by Hector above, if you make changes to the githubHandle syntax, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@kateolenya I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/4796/commits/853dc65894a61130153b5e79c3622ffb5c6d1b9e. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**